group_vars/whosbuying.yml

```diff
@@ -0,0 +1,16 @@
+---
+
+ubuntu_firewall_rules:
+  - rule: allow
+    port: 80
+  - rule: allow
+    port: 443
+
+web_app_name: coffee
+web_app_user: coffee
+web_app_proxy_port: 5001
+web_app_hostnames:
+  - whosebuying.coffee
+  - www.whosebuying.coffee
```

Comment:
This file doesn't need to exist, but we do need to add these two domains to list of `web_app_hostnames` in `group_vars/coffee.yml`.